lib/msal-core/package.json

```diff
@@ -28,50 +28,60 @@
     "clean": "shx rm -rf dist docs lib-commonjs lib-es6",
     "doc": "typedoc --out ./docs ./src/ --gitRevision dev",
     "build:modules": "tsc && tsc -m es6 --outDir lib-es6",
-    "build": "npm run clean && npm run doc && npm run build:modules && npm run lint && npm run build:webpack && npm test",
-    "test": "karma start && jasmine tests/MSALSpec.js",
+    "build": "npm run clean && npm run doc && npm run build:modules && npm run lint && npm run build:webpack && npm run test",
+    "jtest": "typings install && karma start && jasmine-node --nohelpers tests/MSALSpec.js",
```

Comment:
I thought we got rid of typings

Reply:
Could be that the first setup is done bases on the integration branch and dev. we synced to dev yesterday. Will change this. Also This is fallback.

Reply:
This typings call is unnecessary, it doesn't do anything anyway.

Reply:
I have taken out typings in my latest commit.